src/main/java/com/denizenscript/depenizen/bukkit/bridges/LuckPermsBridge.java

```diff
@@ -1,66 +1,80 @@
 package com.denizenscript.depenizen.bukkit.bridges;
 
-import com.denizenscript.depenizen.bukkit.properties.luckperms.LuckPermsPlayerProperties;
-import com.denizenscript.depenizen.bukkit.objects.luckperms.LuckPermsTrackTag;
-import com.denizenscript.depenizen.bukkit.Bridge;
-import com.denizenscript.denizen.objects.PlayerTag;
 import com.denizenscript.denizencore.objects.ObjectFetcher;
-import com.denizenscript.denizencore.tags.TagRunnable;
 import com.denizenscript.denizencore.objects.core.ListTag;
-import com.denizenscript.denizencore.objects.properties.PropertyParser;
-import com.denizenscript.denizencore.tags.Attribute;
-import com.denizenscript.denizencore.tags.ReplaceableTagEvent;
+import com.denizenscript.denizencore.tags.PseudoObjectTagBase;
 import com.denizenscript.denizencore.tags.TagManager;
+import com.denizenscript.depenizen.bukkit.Bridge;
+import com.denizenscript.depenizen.bukkit.objects.luckperms.LuckPermsGroupTag;
+import com.denizenscript.depenizen.bukkit.objects.luckperms.LuckPermsTrackTag;
+import com.denizenscript.depenizen.bukkit.properties.luckperms.LuckPermsPlayerExtensions;
 import net.luckperms.api.LuckPerms;
 import net.luckperms.api.LuckPermsProvider;
-import net.luckperms.api.track.Track;
 
 public class LuckPermsBridge extends Bridge {
 
+    static class LuckPermsTagBase extends PseudoObjectTagBase<LuckPermsTagBase> {
+
+        public static LuckPermsTagBase instance;
+
+        public LuckPermsTagBase() {
+            instance = this;
+            TagManager.registerStaticTagBaseHandler(LuckPermsTagBase.class, "luckperms", (t) -> instance);
+        }
+
+        public void register() {
+
+            // <--[tag]
+            // @attribute <luckperms.list_tracks>
+            // @returns ListTag(LuckPermsTrackTag)
+            // @plugin Depenizen, LuckPerms
+            // @description
+            // Returns a list of all tracks.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "list_tracks", (attribute, object) -> {
+                return new ListTag(luckPermsInstance.getTrackManager().getLoadedTracks(), LuckPermsTrackTag::new);
+            });
+
+            // <--[tag]
+            // @attribute <luckperms.track[<track_name>]>
+            // @returns LuckPermsTrackTag
+            // @plugin Depenizen, LuckPerms
+            // @description
+            // Returns the track from the name given.
+            // -->
+            tagProcessor.registerTag(LuckPermsTrackTag.class, LuckPermsTrackTag.class, "track", (attribute, object, track) -> track);
```

Comment:
Might've been unclear on Discord earlier - this should still be a statement lambda to be consistent with the rest of the tags